index.js

```diff
@@ -75,7 +75,7 @@ var new_client = function(api_key, config) {
       },
       function(error) {
         console.log("[LaunchDarkly] Error: %j", error);
-        cb(error);
+        cb(default_val);
```

Comment:
This was a pretty bad error-- the callback is expecting the value to use, not some random error.